workbench/src/renderer/components/InvestTab/index.jsx

```diff
@@ -208,7 +208,7 @@ class InvestTab extends React.Component {
       return (<div />);
     }
 
-    const logDisabled = !logfile;
+    const logDoesNotExist = !logfile;

```

Comment:
> It did make me think of a case where logfile is defined, but the file does not exist. This could happen if a user deletes the logfile from the workspace after the model run, and then goes back to the list of "Recent Runs", and opens that model tab.

Interesting point. I agree, it seems like that would be a very rare scenario. Still, it makes me wonder if we could refactor the open workspace method so that it doesn't require a logfile at all. You'd know better than I why it was written to be dependent on the logfile, but if there's a way to pass in just the workspace path instead, this could also be a more elegant solution to the issue resolved in this PR. Instead of hiding the Open Workspace button, leaving the user with a total mystery, we could keep the button active, and when it opened the workspace, it would become immediately clear to the user that the workspace is empty (which would still leave them with a mystery, but at least there's a clue as well).

Reply:
I completely agree, de-coupling this button from the logfile makes a lot of sense. And opening up to an empty workspace sounds just fine. There's still probably a case where the workspace folder never exists and we need some error-handling. 

I can't say for sure why it is the way it is. I guess the logfile is the most reliable artifact left behind by running an invest model, so it's possible that we were just avoiding some redundancy by not storing both the logfile path _and_ the workspace path in an `InvestJob` instance, for example.

Also, the available `electron.shell` APIs must have been a factor: https://www.electronjs.org/docs/latest/api/shell
Maybe `shell.openPath` would work instead of `showItemInFolder` if given a path to a directory.

Reply:
I tested `shell.openPath`, and it does open a folder in Finder, if you pass in a path to a directory (whereas if you pass in a file path, it will actually open the file). I'm going to try going down that (ahem) _path_ and put this PR on hold. If the other approach works, I'll abandon this PR; if I run into a snag, we can merge this PR as a stopgap.